.github/actions/get-workflow-job-id/action.yml

```diff
@@ -26,6 +26,20 @@ runs:
         retry_wait_seconds: 30
         command: |
           set -eux
-          python3 -m pip install requests==2.26.0
-          GHA_WORKFLOW_JOB_ID=$(python3 .github/scripts/get_workflow_job_id.py "${GITHUB_RUN_ID}" "${RUNNER_NAME}")
+
+          if [[ -z "${CONDA_RUN:-}" ]]; then
+            # If no conda has been setup, try to use system python
+            python3 -m pip install requests==2.26.0
+            GHA_WORKFLOW_JOB_ID=$(python3 .github/scripts/get_workflow_job_id.py "${GITHUB_RUN_ID}" "${RUNNER_NAME}")
+          else
+            # Windows miniconda, unfortunately, doesn't have a python3 binary, but
```

Comment:
Is there a reference to the doc somewhere saying this is not the case? All the PEPs I've raid state otherwise - if Python-3 is installed, then `python3` executable should be there, but `python` is optional

Reply:
Oh, I look into the C:\Jenkins\Miniconda3 where it's installed but couldn't find a python3 binary anywhere.  My guess is that python3 alias will only pop up after conda is activated.

Reply:
Hmm, it should be other way around (i.e. `python` is an alias to either `python3` or `python2`)
But again, perhaps the better solution would be to create this alias if it's missing in `setup-win` and then use `python3` throughout the scripts?

Reply:
Yeah, I'll update the PR to use this approach

Reply:
> Hmm, it should be other way around (i.e. `python` is an alias to either `python3` or `python2`) But again, perhaps the better solution would be to create this alias if it's missing in `setup-win` and then use `python3` throughout the scripts?

It's clearer now after reading the way Miniconda is packaged in https://docs.conda.io/en/latest/miniconda.html.  There are two flavors `Miniconda3` and `Miniconda2` and they can both be installed in different directories.  Each will have one built-in python executable, and it can be Python 3 or 2 depending on which installation package is used.

As we are using `Miniconda3` for Windows, creating a `python3` softlink pointing to `python` works.